apps/sms/js/notify.js

```diff
@@ -56,16 +73,22 @@
 
   var Notify = {
     ringtone: function notification_ringtone() {
-      if (settings['audio.volume.notification'] &&
-          settings['notification.ringtone']) {
-        ringtone(settings['notification.ringtone']);
-      }
+      return getSettings(
+        ['audio.volume.notification', 'notification.ringtone']
```

Comment:
Looks like we'll reuse these settings keys frequently. Maybe define these string to global scope?